codersdk/externalauth.go

```diff
@@ -33,9 +34,10 @@ func (e EnhancedExternalAuthProvider) Git() bool {
 }
 
 const (
-	EnhancedExternalAuthProviderAzureDevops EnhancedExternalAuthProvider = "azure-devops"
-	EnhancedExternalAuthProviderGitHub      EnhancedExternalAuthProvider = "github"
-	EnhancedExternalAuthProviderGitLab      EnhancedExternalAuthProvider = "gitlab"
+	EnhancedExternalAuthProviderAzureDevops      EnhancedExternalAuthProvider = "azure-devops"
+	EnhancedExternalAuthProviderAzureDevopsEntra EnhancedExternalAuthProvider = "azure-devops-entra"
```

Comment:
Comment the difference here? I know you have it in the docs, but the internal devs don't read those very much :smile:

Reply:
Fair enough! Added a comment